isgx_page_cache.c

```diff
@@ -479,7 +479,7 @@ void isgx_free_epc_page(struct isgx_epc_page *entry,
 	}
 
 	if (flags & ISGX_FREE_EREMOVE)
-		BUG_ON(isgx_eremove(entry));
+		isgx_eremove(entry);
```

Comment:
Yes it is there. The display here is confusing; if you look at the tab 'files changed', you will see two file changes, the second one is the pr_err call.